src/app/youtube/directives/border-bottom-color.directive.ts

```diff
@@ -0,0 +1,40 @@
+import { Directive, ElementRef, Renderer2, Input, OnInit } from '@angular/core';
+import { DatesLagValue, DatesLagColor } from '../_constants/datesLag.constants';
+
+@Directive({
+    selector: '[appBorderBottomColor]'
+})
+export class BorderBottomColorDirective implements OnInit {
```

Comment:
Why did you decide to use directive instead of just css styling through classes? Your solution is seemed overhead